pkg/skaffold/config/util_test.go

```diff
@@ -498,11 +498,11 @@ func TestShouldDisplayPrompt(t *testing.T) {
 			},
 		},
 		{
-			description: "should display prompt when last prompted is before 2 weeks",
+			description: "should display prompt when last prompted is older than 3 months",
 			cfg: &ContextConfig{
 				Survey: &SurveyConfig{
 					DisablePrompt: util.BoolPtr(false),
-					LastPrompted:  "2019-01-10T00:00:00Z",
+					LastPrompted:  "2019-09-10T00:00:00Z",
```

Comment:
This date is *after* the current time (2019-01-30T12:04:05Z).
```suggestion
					LastPrompted:  "2018-09-10T00:00:00Z",
```